src/main/frontend/app/containers/ProductForm.js

```diff
@@ -158,52 +158,53 @@ class ProductForm extends Component {
     }
 
     return (
-      <form onSubmit={this.persistProduct}>
-        {errorDiv}
-        <FieldInput
-          label="Name: "
-          name="name"
-          type="text"
-          onChange={this.onChange}
-          value={name}
-        />
-        <br />
-        <FieldInput
-          label="Price: "
-          name="price"
-          type="number"
-          onChange={this.onChange}
-          value={price}
-        />
-        <br />
-        <FieldInput
-          label="Description: "
-          name="description"
-          type="text"
-          onChange={this.onChange}
-          value={description}
-        />
-        <br />
-        <FieldInput
-          label="Product Url: "
-          name="url"
-          type="text"
-          onChange={this.onChange}
-          value={url}
-        />
-        <br />
-        <FieldInput
-          label="Image Url: "
-          name="imageUrl"
-          type="text"
-          onChange={this.onChange}
-          value={imageUrl}
-        />
-        <br />
-        <div>
-          <input type='submit' value='Add product' />
-        </div>
-      </form>
+      <div className="container">
+        <h1>Add a New Product</h1>
+        <form onSubmit={this.persistProduct}>
+          {errorDiv}
+          <FieldInput
+            label="Name: "
+            name="name"
+            type="text"
+            onChange={this.onChange}
+            value={name}
+          />
+          <br />
```

Comment:
can we use css and some different markup here to eliminate the need for a `<br />` tag